src/silvimetric/commands/info.py

```diff
@@ -1,37 +1,57 @@
-import json
+import numpy as np
+from uuid import UUID
 
-from ..resources import Storage
+from ..resources import Storage, Bounds
 
-def info(tdb_dir, history):
+def check_values(start_time, end_time, bounds, name):
+    if start_time is not None and not isinstance(start_time, np.datetime64):
+        raise TypeError(f'Incorrect type of "start_time" argument.')
+    if end_time is not None and not isinstance(end_time, np.datetime64):
+        raise TypeError(f'Incorrect type of "end_time" argument.')
+    if bounds is not None and not isinstance(bounds, Bounds):
+        raise TypeError(f'Incorrect type of "bounds" argument.')
+    if name is not None:
+        try:
+            UUID(name)
+        except:
+            raise TypeError(f'Incorrect type of "name" argument.')
+
+def info(tdb_dir, start_time:np.datetime64=None, end_time:np.datetime64=None,
+          bounds:Bounds=None, name:str=None):
     """Print info about Silvimetric database"""
+    check_values(start_time, end_time, bounds, name)
+
     with Storage.from_db(tdb_dir) as tdb:
+        if start_time is None:
```

Comment:
Which is the Python way?

```
if foo is None:
    # something

if not foo:
    # something
```

Reply:
`if foo is None` is the correct one, from what I've read